File: python/wc.py
#!/usr/bin/python

#readin and filter
txt = [c for c in open('lorem.txt').read().lower() if c.isalpha() or c==' ']
 
#groupBy in a dictionary
wc = dict()
for w in ''.join(txt).split():
  wc[w] = wc.setdefault(w, 0) + 1
 
#output
for k,v in wc.iteritems():
   print('{0}\t{1}'.format(k,v))

File: R/wc.R
#!/usr/bin/Rscript

#read in and clean
x = scan("lorem.txt", what="", sep=" ")
x.clean = gsub("\\W+","",tolower(x))
 
#word count
wc = table(x.clean)
 
#output to stdout
print(as.data.frame(wc), row.names=rep("", nrow(wc)))

